File: qc-application/src/main/java/com/eager/questioncloud/QCApplication.java
package com.eager.questioncloud;


import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableAsync;

@SpringBootApplication
@EnableAsync
public class QCApplication {
    public static void main(String[] args) {
        SpringApplication.run(QCApplication.class, args);
    }
}

File: qc-application/src/main/kotlin/com/eager/questioncloud/QCApplication.kt
package com.eager.questioncloud

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableAsync
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EnableAsync
@EnableScheduling
class QCApplication

fun main(args: Array<String>) {
    runApplication<QCApplication>(*args)
}
